src/utils/openai-utils.ts

```diff
@@ -15,60 +15,33 @@ export const openAiTools: AssistantTool[] = [
   {
     type: "function",
     function: {
-      name: "get_attributes",
-      description: "Get a list of all attributes in a dataset",
-      strict: true,
+      name: "create_request",
+      description: "Create a request to get data from CODAP",
```

Comment:
Not just get data, right?  It could create/alter/delete components as well.

Reply:
Yes -- it can use all of the available CODAP API create/update/delete endpoints.